virtwho/parser.py

```diff
@@ -456,9 +457,14 @@ def parse_options():
     This function parses all options from command line and environment variables
     :return: Tuple of logger and options
     """
+    # These options are deprecated
+    DEPRECATED_OPTIONS = ['log_per_config', 'log_dir', 'log_file', 'reporter_id']
 
     # Read command line arguments first
     cli_options, defaults = parse_cli_arguments()
+    for option in DEPRECATED_OPTIONS:
+        if option in cli_options and not cli_options[option] == defaults[option]:
+            print("The option --%s is deprecated and will be removed in the next release." % option)
```

Comment:
I would add something to the effect of "please see 'man virt-who-config' for details on adding a configuration section"